doc/version_history.rst

```diff
@@ -2,6 +2,14 @@
 Version History
 ===============
 
+v0.8.3
```

Comment:
I think the changes are large enough for a new minor release... you are converting some methods to coroutines which I think are pretty big changes..

Reply:
Update the version in commit: 486bfd3. Thanks!